README.md

```diff
@@ -1,15 +1,15 @@
 #### Table of Contents
 
-1. [Description](#description)
-2. [Setup - The basics of getting started with rook](#setup)
-3. [Usage - Configuration options and additional functionality](#usage)
-   * [Tasks](#tasks)
-   * [Validating and unit testing the module](#validating-and-unit-testing-the-module)
-4. [Reference - An under-the-hood peek at what the module is doing and how](#reference)
-   * [Classes](#classes)
-   * [Parameters](#parameters)
-5. [Limitations - OS compatibility, etc.](#limitations)
-6. [Development - Guide for contributing to the module](#development)
+- [Description](#Description)
+- [Setup](#Setup)
+- [Usage](#Usage)
+  - [Tasks](#Tasks)
+  - [Validating and unit testing the module](#Validating-and-unit-testing-the-module)
+- [Reference](#Reference)
+  - [Classes	For information on the classes and types, see the REFERENCE.md.](#Classes-For-information-on-the-classes-and-types-see-the-REFERENCEmd)
```

Comment:
I don't think the classes information is necessary in the table of contents as you should point directly to REFERENCE.md from line 81 in the README

Reply:
@carabasdaniel Thank you Daniel